src/test/scala/mesosphere/marathon/api/v2/json/AppDefinitionTest.scala

```diff
@@ -600,12 +600,32 @@ class AppDefinitionTest extends UnitTest with ValidationTestLike {
 
     }
 
+    "Validate URIs with query strings correctly" in {
+      val json =
+        """
+      {
+        "id": "app-with-fetch",
+        "cmd": "brew update",
+        "uris": ["http://example.com/file1.tar.gz?foo=10&bar=meh", "http://example.com/file?foo=10&bar=meh"]
+      }
+      """
```

Comment:
Why not use strip margin here?

Reply:
+1

Reply:
I would stay consistent to the other tests in this suite and not change style nor enhance scope for this ticket. WDYT @meichstedt @jeschkies ?